client-service/src/main/java/com/lohika/jclub/client/ClientController.java

```diff
@@ -0,0 +1,41 @@
+package com.lohika.jclub.client;
+
+import com.lohika.jclub.storage.client.Apartment;
+import com.lohika.jclub.storage.client.StorageServiceClient;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PagedResourcesAssembler;
+import org.springframework.hateoas.PagedResources;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@RestController
+public class ClientController {
+
+    @Autowired
+    private StorageServiceClient storageServiceClient;
+
+    @GetMapping("/apartments")
+    public PagedResources<Apartment> getApartments() {
+        PagedResources<Apartment> list = storageServiceClient.list();
+        Collection<Apartment> content = list.getContent();
+        return new PagedResources<>(content, list.getMetadata());
+        //TODO wrap appratments with resources. generate links to client service.
+    }
+
+
+    @GetMapping("/apartmentsPaged")
+    public PagedResources<Apartment> getApartmentsPaged(Pageable pageable,
+                                                        PagedResourcesAssembler assembler) {
```

Comment:
Which line length do you use ? Looks like you may move up this argument and save pretty code style.